lib/github.com/diku-dk/linalg/linalg.fut

```diff
@@ -45,11 +127,15 @@ module type field = {
   val **: t -> t -> t
 
   val neg: t -> t
-  val <: t -> t -> bool
+  -- val inf: t
```

Comment:
Is this intentional?

Reply:
I will remove that, it was left-over from some point earlier in development but it must not have been needed.